app/data_providers/snowplow/snowplow_corpus_slate_tracker.py

```diff
@@ -43,6 +46,7 @@ def _get_corpus_slate_entity(self, corpus_slate: CorpusSlateModel) -> SelfDescri
             schema=self.snowplow_config.CORPUS_SLATE_SCHEMA,
             data={
                 'corpus_slate_id': corpus_slate.id,
+                'recommended_at': int(corpus_slate.recommended_at.timestamp()),
```

Comment:
Is our pattern to always encode dates as epoch timestamps?

Reply:
Yes. I'll start documenting Snowplow patterns like this on Confluence per our retro discussion on Friday.

Decision to settle on epoch was here: https://github.com/Pocket/spec/pull/152